OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -302,6 +405,20 @@ ScrollItemWidget BasicStats(Player player)
 			return template;
 		}
 
+		void SetupPlayerColor(Player player, ScrollItemWidget template, ColorBlockWidget colorBlockWidget, GradientColorBlockWidget gradientColorBlockWidget)
+		{
+			var color = Color.FromArgb(128, player.Color.R, player.Color.G, player.Color.B);
+			var hoverColor = Color.FromArgb(230, player.Color.R, player.Color.G, player.Color.B);
+
+			colorBlockWidget.GetColor = () => template.EventBounds.Contains(Viewport.LastMousePos) ? hoverColor : color;
+
+			gradientColorBlockWidget.GetTopLeftColor = () => template.EventBounds.Contains(Viewport.LastMousePos) ? hoverColor : color;
```

Comment:
The issue with using `EventBounds` here is that the hover effect triggers when using the tab dropdown.

This can be fixed using something like:
```csharp
var isMouseOver = new CachedTransform<Widget, bool>(w => w == template || template.Children.Contains(w));
colorBlockWidget.GetColor = () => isMouseOver.Update(Ui.MouseOverWidget) ? hoverColor : color;
gradientColorBlockWidget.GetTopLeftColor = () => isMouseOver.Update(Ui.MouseOverWidget) ? hoverColor : color;
gradientColorBlockWidget.GetBottomLeftColor = () => isMouseOver.Update(Ui.MouseOverWidget) ? hoverColor : color;
gradientColorBlockWidget.GetTopRightColor = () => isMouseOver.Update(Ui.MouseOverWidget) ? hoverColor : Color.Transparent;
gradientColorBlockWidget.GetBottomRightColor = () => isMouseOver.Update(Ui.MouseOverWidget) ? hoverColor : Color.Transparent;
```
then adding `Ui.ResetTooltips()` after `a()` in `createStatsOption` and removing `template.IgnoreMouseOver = false` from  `ProductionStats` (this doesn't seem to be needed?).

I suspect you will also need to add `template.IgnoreChildMouseOver = false` to `SupportPowerStats` for the support power tooltips to work too!